round4/panchel.md

```diff
@@ -0,0 +1,95 @@
+3. Risk Mitigation: The ability to set custom stop-loss conditions and automate trade executions helps users effectively manage risk and protect their investments, even during periods of market volatility or when unable to actively monitor positions.
+4. Centralized View: The extension provides a centralized view of the user's trading activities across multiple wallets and chains, enabling better decision-making and portfolio management.
+5. Convenience and Accessibility: As a Chrome extension, ProfitPeak offers a convenient and easily accessible solution, eliminating the need for additional software installations or dedicated applications running in the background.
+
+## User Stories
+
+1. As a crypto trader with limited time, I want to be able to track all of my trades.
+2. As a portfolio manager using multiple chains and providers, I want to be able to create reports for executives on aggregated performance.
+3. As a trader with a sleep schedule, I want to set up certain stop-losses on risky trades since the market never sleeps.
+4. As an inexperienced crypto trader, I want a platform that helps me analyze my crypto exchanges and offer detailed visualizations to help me understand my trading efficiency.
+5. As a crypto trader, I want a centralized platform to quickly see all my past trades and what they are worth now so that I can track my progress.
+6. As a crypto trader looking to improve, I want to analyze the performance of my past trades so that I can identify successful strategies and areas for improvement.
+7. As a new trader, I want an easy way to track and visualize my progress so that I can keep learning.
+8. As a cryptocurrency enthusiast, I want to be able to manage a wide range of cryptocurrencies and tokens, so that I can manage all my assets efficiently.
+9. As a trader, I want the extension to calculate my profit and loss (PnL) for each trade, so that I can accurately evaluate the success of my trading strategies.
+10. As a crypto investor, I want to easily generate tax reports for my crypto transactions so that I can comply with regulations without hassle.
+11. As an avid crypto trader, I want alerts on the values of my trades so that I can make informed decisions about my crypto investments.
+12. As a beginner trader, I want tips and suggestions on when to buy and sell so that I can keep learning to become a better trader.
+13. As a crypto trader, I want a tool/platform that aggregates my trades across various (unique) exchanges.
+14. As a portfolio manager, I want a one stop shop to see all my transactions so that I can quickly manage them all at once. 
+15. As a cryptocurrency educator, I want to provide clear reports on transactions so that I can engage my students and help them develop a deeper understanding of blockchain technology and trading strategies.
+
```

Comment:
Overall sounds promising! I think it'd be good to think about what core functionalities makes this Chrome extension better than adding widgets from Coinbase and Fidelity onto your home page on your phone. If you're able to achieve that, I think this extension could be great.

Looking forward to using the extension!